webapp/pom.xml

```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```

Comment:
Is there a reason to go to another 6.* version of Jetty, is it possible to go to a recent stable version of Jetty 8? One reason I ask is that core is now building against Servlet 3.0 and there have been discussions about utilizing that functionality, so an upgrade of Jetty to 8 would be ideal if it works out.